### PR TITLE
Added example of lookup value selection and MFA

### PR DESCRIPTION
**Pull request by crisfervil**
I'm not sure if this is appropriate, given that this is a basic example, but there it goes anyway. Feel free to reject it if not!

### CODE REVIEW

File: tests/app.spec.js [hunk 1]
```diff
@@ -38,9 +44,21 @@ test('create a contact', async ({ page }) => {
...
+    const lookupFieldTextValue = 'Northwind traders';
+    const clientLookup = page.locator(`[aria-label="${lookupFieldLabel}, Lookup"]`);
+
+    await clientLookup.hover();
+    await clientLookup.click();
+    await clientLookup.fill('');
+    await clientLookup.type(lookupFieldTextValue);
+
+    await page.click('[aria-label="Lookup results"]>li');
+
```
Comment (ryanspain) [inline tests/app.spec.js:58]: Works on my machine! Thanks for adding this.

chrome_vLpOui3J2j

**Review by ryanspain — CHANGES_REQUESTED** [review]

File: tests/app.spec.js [hunk 2]
```diff
@@ -38,9 +44,21 @@ test('create a contact', async ({ page }) => {
...
     // Click on the Save button in the command bar
-    await page.click('[aria-label="Save (CTRL+S)"]')
-    await page.waitForNavigation(),
+    await page.click('[aria-label="Save (CTRL+S)"]');
+    await page.waitForNavigation();
```
Comment (ryanspain) [inline tests/app.spec.js:61]: Thanks for spotting that. JavaScript is too forgiving 😄

File: tests/app.spec.js [hunk 3]
```diff
@@ -12,6 +12,12 @@ test.beforeEach(async ({ page }) => {
     await page.fill('input[name="passwd"]', config.password);
     await page.click('input[value="Sign in"]');
     await page.waitForNavigation();
+    // Click on the remember me button
+    await page.click('input[value="Yes"]');
+    await page.waitForNavigation();
+    // wait until the URL does not contain the word 'login'
+    // useful for scenarios where MFA is active. It will give you 60 seconds to approve it.
+    await page.waitForURL(/^(?:(?!login).)*$/,{timeout:60000});
```
[the remark this answers is not in the document]
Comment (ryanspain) [reply tests/app.spec.js:20]: I merged your changes (accidentally) with this and manually dropped the interaction with the **Stay signed in** screen 👍

**Comment by crisfervil** [comment]
Thanks @ryanspain !! You even added screen recordings!

**Pull request merged**